lib/util/xpcapng.c

```diff
@@ -0,0 +1,630 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ *  Description:
+ *      Simple PcapNG library developed from scratch as no library existed that
+ *      met the requirements for xdpdump. It can also be used by other XDP
+ *      applications that would like to capture packets for debugging purposes.
+ *
+ *  Notes:
```

Comment:
Notes? :)

Reply:
Ooops, had some notes which I removed, forgot the header. Wil remove in the next update